src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.test.tsx

```diff
@@ -212,9 +212,12 @@ describe('FixPhoneNumbers-Home', () => {
     userEvent.click(getByText('Yes'));
 
     await waitFor(() => {
-      expect(mockEnqueue).toHaveBeenCalledWith(`Phone numbers updated!`, {
-        variant: 'success',
-      });
+      expect(mockEnqueue).toHaveBeenCalledWith(
+        `Successfully updated phone numbers for Simba Lion`,
+        {
+          variant: 'success',
+        },
+      );
```

Comment:
Maybe here is a good place to ensure stale data is not stored and sent to the server.

Reply:
Tests written in contact.test.tsx